src/app/shared/dialogs/dialogs-list.component.ts

```diff
@@ -45,7 +45,7 @@ export class DialogsListComponent implements AfterViewInit {
     this.tableData.data = this.data.tableData;
     this.tableColumns = this.data.columns;
     this.disableRowClick = this.data.disableSelection || false;
-    this.emptySubmit = this.data.selectionOptional || false;
+    this.emptySubmit = this.data.selectionOptional || true;
```

Comment:
@BhargaviNadendla you should not change it here. It is default value if you do not pass it during call from other component. You should pass `selectionOptional` form course-step during call for dialoglist